src/main/java/com/artipie/rpm/Cli.java

```diff
@@ -34,76 +38,96 @@
  * Cli tool main class.
  *
  * @since 0.6
+ * @todo #89:30min Cli options are not parsed correctly. getOptionValue method always return
+ *  default value for both options, so it's not possible to override them. But the tool is working
+ *  correctly. To build it use `cli` maven profile, see the README.
  */
 public final class Cli {
-    /**
-     * Naming policy option object.
-     */
-    private static final Option NAMING_OPTION = Option.builder("np")
-        .longOpt("naming-policy")
-        .desc("(optional, default simple) configures NamingPolicy for Rpm")
-        .hasArg()
-        .build();
-
-    /**
-     * Digest option object.
-     */
-    private static final Option DIGEST_OPTION = Option.builder("d")
-        .longOpt("digest")
-        .desc("(optional, default sha256) configures Digest instance for Rpm")
-        .hasArg()
-        .build();
-
-    /**
-     * Options object.
-     */
-    private static final Options OPTIONS = new Options();
-
-    static {
-        Cli.OPTIONS.addOption(Cli.NAMING_OPTION);
-        Cli.OPTIONS.addOption(Cli.DIGEST_OPTION);
-    }
 
     /**
-     * Command line parser object.
+     * Rpm tool.
      */
-    private static final CommandLineParser PARSER = new DefaultParser();
+    private final Rpm rpm;
 
     /**
      * Ctor.
+     * @param rpm Rpm instance
      */
-    private Cli() {
+    private Cli(final Rpm rpm) {
+        this.rpm = rpm;
     }
 
     /**
      * Main method of Cli tool.
      *
      * @param args Arguments of command line
      * @throws ParseException if parsing failed
-     * @todo #68:30min start execution of the command
-     *  you can use cmd.getArgs() to get args as array of strings
-     *  you would expect something like ["update", "./package.rpm"]
-     *  use this link for documentation
-     *  https://commons.apache.org/proper/commons-cli/usage.html
+     * @checkstyle IllegalCatchCheck (70 lines)
+     * @checkstyle LineLengthCheck (50 lines)
      */
-    @SuppressWarnings("ProhibitPublicStaticMethods")
-    public static void main(final String[] args) throws ParseException {
-        @SuppressWarnings("PMD.UnusedLocalVariable")
-        final CommandLine cmd = parseCommand(args);
+    @SuppressWarnings(
+        {
+            "PMD.SystemPrintln", "PMD.AvoidCatchingGenericException",
+            "PMD.DoNotCallSystemExit", "PMD.AvoidDuplicateLiterals"
+        }
+    )
+    public static void main(final String... args) throws ParseException {
+        final CommandLine cli = new DefaultParser().parse(
```

Comment:
@g4s8 how about we extract parsing of cli arguments to a separate class that will provide variables  `naming, digest, path` required for instantiation of `Rpm`? It would be easier to test and maintain that functionality.